docs/releases/unreleased.md

```diff
@@ -0,0 +1,20 @@
+# OPA Core Module version XXX
+
+SIGHUP team maintains this module updated and tested. That is the main reason why we worked on this new release.
+With the Kubernetes 1.21 release, it became the perfect time to start testing this module against this Kubernetes
+release.
+
+Continue reading the [Changelog](#changelog) to discover them:
+
+## Changelog
+
+- Add a Constraint Template to protect namespaces for being deleted.
```

Comment:
Can we explain here or (maybe better) in the readme how to use the new rule? The annotations, etc.